deploy/gateway/templates/NOTES.txt

```diff
@@ -15,7 +15,7 @@ Admin Console URL - {{ printf "https://%s.%s" .Values.twingate.network (include
 
 1. The Gateway is accessible via its LoadBalancer IP. It may take a few minutes for the IP address to be available.
```

Comment:
Let's also update the text here to reflect this change e.g. 
> The Gateway is accessible via its LoadBalancer IP or hostname. It may take a few minutes for the address to be available.